### DISCUSSION TITLE
Show tab titles in Live Preview

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9153] [Ideas] [answered]
> Created by: Sander Tiekstra (sandertiekstra@gmail.com) on 2015/03/25 14:09:22 +0000
> Votes at time of UserVoice import: 8

-----------------------------------

The field layout titles are used as tabs in the 'normal' view, but are missing in the Live Preview. Just showing the titles between the field groups would be a great addition. Nice-to-have: position the titles as fixed when scrolling, as the Redactor toolbar does.

**Comment by angrybrad** [comment]
> Posted by Stuart McCoy (stuart@meltingcrayons.com) on 2015/03/11 01:51:01 +0000
-----------------------------------

I&#39;d also add another possibility. Convert the tabs to accordion panels that open and stay open unless closed. The first panel would obviously be opened when Live Preview is initiated.

**Comment by leevigraham** [comment]
I've just trained two clients this week and they love live preview. Accordions would be awesome to help simplify / organise fields.

**Comment by angrybrad** [comment]
Related: https://github.com/craftcms/cms/issues/969

**Comment by TomDeSmet** [comment]
This would be so awesome! I just created a content builder with the Neo plugin in its own tab. But there are 5 other tabs with information and matrix fields and in live preview, although I find it gorgeous how my content builder works in live preview :), it's a hell of a clutter with all the other tab fields in there as well.

**Comment by leevigraham** [comment]
@TomDeSmet  Can you post some screenshots of the normal and live preview editing UI with your content builder?

**Comment by harryfk** [comment]
Same setup here … an accordion would be fantastic.

**Comment by johnwbaxter** [comment]
Yeah, i'm with this request!

I like to lay my fields out in tabs to make it logical for the author, and then they use live preview (which they do cos they love it) and then all that organisation is out the window.

Accordions sound nice!

**Comment by brandonkelly** [comment, marked as answer]
Live Preview is gaining multi-tab support in Craft 5.2 🎉 (#15112)

**Reply by brandonkelly** [reply]
Craft 5.2.0-beta.1 is out with those changes.
